scripts/build_backend.py

```diff
@@ -16,18 +16,20 @@
 # limitations under the License.
 # ===============================================================================
 
+import logging
 import multiprocessing
 import os
 import platform as plt
 import subprocess
 import sys
-from distutils import log
-from distutils.sysconfig import get_config_var, get_python_inc
 from math import floor
 from os.path import join as jp
+from sysconfig import get_config_var, get_paths
 
 import numpy as np
 
+logger = logging.getLogger("sklearnex")

```

Comment:
Just make sure that all logs works as expected, nothing missed.
Compare it with old one please.